summoner.cabal

```diff
@@ -0,0 +1,68 @@
+name:                summoner
+version:             0.0.0
+description:         Tool for creating completely configured production Haskell projects.
+synopsis:            Tool for creating completely configured production Haskell projects.
+homepage:            https://github.com/kowainik/summoner
+bug-reports:         https://github.com/kowainik/summoner/issues
+license:             MPL-2.0
+license-file:        LICENSE
+author:              Kowainik
+maintainer:          xrom.xkov@gmail.com
+copyright:           2018 Kowainik
+category:            CLI, CLI Tool, Development
+build-type:          Simple
+extra-doc-files:     README.md
+cabal-version:       >=1.24
+tested-with:         GHC == 8.2.2
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Summoner
+                         Summoner.Ansi
+                         Summoner.CLI
+                         Summoner.Default
+                         Summoner.License
+                         Summoner.Process
+                         Summoner.Project
+                         Summoner.ProjectData
+                         Summoner.Question
+                         Summoner.Template
```

Comment:
I like your current module structure! Really helps to navigate between things :heart_eyes: